PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -5041,6 +5044,16 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
     return kFALSE;
   }
 
+  if (fSpecialTrigger == 6){
+    AliCaloTriggerMimickHelper* tempMimickHelper = 0x0;
+    tempMimickHelper = (AliCaloTriggerMimickHelper*) (AliAnalysisManager::GetAnalysisManager()->GetTask(Form("CaloTriggerHelper_%s", GetCutNumber().Data()) ));
+    if (tempMimickHelper){
+      return tempMimickHelper->GetEventChosenByTrigger();
+    } else {
+      return kFALSE;
+    }
+  }
+
```

Comment:
If this is the only way that trigger mimicking should work for PHOS, then please make it an AliFatal if the helper is not found.